api-admin/src/main/java/bio/terra/pearl/api/admin/service/siteContent/SiteContentExtService.java

```diff
@@ -33,6 +33,8 @@ public Optional<SiteContent> get(
 
   public SiteContent create(
       String portalShortcode, String stableId, SiteContent siteContent, AdminUser user) {
+    // TODO: Updating content for one version shouldn't destroy all other localizedSiteContents
+    //      We should carry forward the existing and unmodified localizedSiteContents
```

Comment:
We'll either need to solve this in the backend or when we update the admin UX, but right now if you update the site content for one language, it doesn't allow you to bring the other languages into the new version of the SiteContent, so you'll lose them

Reply:
Yeah,I think this method should create exactly what's passed into it via the SiteContent argument, since for all the backend knows the user wants to remove a language (and that's also how our survey editor works).  So I'd say we fix this on the frontend